yarn.lock

```diff
@@ -1506,24 +1516,26 @@ balanced-match@^1.0.0:
   resolved "https://registry.yarnpkg.com/balanced-match/-/balanced-match-1.0.0.tgz#89b4d199ab2bee49de164ea02b89ce462d71b767"
   integrity sha1-ibTRmasr7kneFk6gK4nORi1xt2c=
 
-base-x@3.0.4, base-x@^3.0.2:
+base-x@3.0.4:
   version "3.0.4"
   resolved "https://registry.yarnpkg.com/base-x/-/base-x-3.0.4.tgz#94c1788736da065edb1d68808869e357c977fa77"
   integrity sha512-UYOadoSIkEI/VrRGSG6qp93rp2WdokiAiNYDfGW5qURAY8GiAQkvMbwNNSDYiVJopqv4gCna7xqf4rrNGp+5AA==
   dependencies:
     safe-buffer "^5.0.1"
 
+base-x@^3.0.2:
+  version "3.0.5"
```

Comment:
Hmm, this also has 2 `base-x` versions, but over here tests still pass...?

Once the `resolutions` block merges to JS SDK, maybe try `yarn install` / `yarn upgrade` again and see if it avoids this?

Reply:
Hmm - I've yarn installed, upgraded, removed node modules and reinstalled and the lockfile come out the same. Even putting an equivalent `resolutions` entry in riot-web's package.json doesn't affect it. However I still only have one version of base-x and that is 3.0.4, so looks like it's fine? Either way I think the only alternative would be to hand-edit the file.

Reply:
Hmm, curious. I suppose if you've only got one version in practice, then maybe it's okay. Seems fine to defer until we find a reason it's breaking something.